playbooks/appsemblerPlaybooks/figures.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+# This role is based on a temporary hack
+# This should configure figures in a prod instance but that's not the correct method
+
+
+- name: Install figures on Gingko
+  hosts: edxapp-server
+  become: True
+  become_method: sudo
+  gather_facts: True
+  vars_files:
+    - ../roles/common_vars/defaults/main.yml
+    - ../roles/edxapp/defaults/main.yml
+  roles:
+    - figures
```

Comment:
I think this role should be a default role inside `ginkgo_basic.yml` playbook instead of having its own playbook. This role should be guarded with a variable to prevent accidental deployment.

```yaml
  roles:
    ...
    - role: figures
      when: FIGURES_INSTALL|default(False)
    ...
```

Reply:
@OmarIthawi I wasn't sure about this. But since we're going to enable it for all Enterprise customers sooner or later, I think it's a good idea.